webpack.config.npm.js

```diff
@@ -156,11 +156,11 @@ const parserConfig = Object.assign({}, defaultConfig, {
     dasksqlAutocompleteParser: [`${JS_ROOT}/parse/sql/dasksql/dasksqlAutocompleteParser.js`],
     dasksqlSyntaxParser: [`${JS_ROOT}/parse/sql/dasksql/dasksqlSyntaxParser.js`]
   },
-  performance: {
-    hints: false,
-    maxEntrypointSize: 1500000,
-    maxAssetSize: 1500000
-  },
+  // performance: {
```

Comment:
I'm not sure this needs to be commented out, I can check if this can be reverted